controllers/clusters/kafkaconnect_controller_test.go

```diff
@@ -58,10 +59,26 @@ var _ = Describe("Kafka Connect Controller", func() {
 					return false
 				}
 
-				return kafkaConnect.Status.ID == openapi.CreatedID
+				return kafkaConnect.Status.ID != ""
 			}).Should(BeTrue())
 
 			<-done
+
+			By("creating secret with the default user credentials")
+			secret := kafkaConnect.NewDefaultUserSecret("", "")
+			secretNamespacedName := types.NamespacedName{
+				Namespace: secret.Namespace,
+				Name:      secret.Name,
+			}
```

Comment:
is that OK to create a `defaultUserSecret` with empty creds?

Reply:
I use it only for getting a secret with an already configured namespaced name for the specified cluster and nothing else.